truffleruby/src/main/ruby/core/rbconfig.rb

```diff
@@ -85,7 +85,7 @@ module RbConfig
 
   if ruby_home
     libdir = "#{ruby_home}/lib"
-    bindir = "#{libdir}/bin"
+    bindir = "#{ruby_home}/bin"
```

Comment:
The current bindir is causing me issues when testing gems, so I hope this line change goes in soon.